code/src/main/webapp/CSS/style.css

```diff
@@ -1,7 +1,15 @@
-body, html {
+html {
   color: #222;
   height: 100%;
+}
+
+body {
```

Comment:
Can you please move these properties to a div? We generally try not to style the body element (though we sometimes set its margin to 0).

Reply:
I saw your comment about not having css code in body, however I did that to improve the transition between background images. Prior to that, changing pages would unload the landing image and load in the other image which caused the background to blink white between loads. I tried to add the image to a div but when I did that the landing background was overwritten by the other background.

Reply:
Why couldn't we have a parent div that lives within the body that accomplishes the same thing? So like

<body><div id="page-wrapper">...elements that are in the body directly today </div></body>

That seems like it would almost certainly work.

Reply:
To clarify my previous comment, I added the non landing background to wrapper-background div and it overwrote the landing background in the landing page.

Reply:
Sorry, edited my comment, hopefully that is clearer.

Reply:
wrapper-background is a the parent div so I don't see how adding another parent div would be any different.

Reply:
fixed it, by specifying that the landing background be applied to #wrapper-background and .landing-background it worked.